common.py

```diff
@@ -3,7 +3,7 @@
 
 
 FT_BASE_IMG = "quay.io/wizhao/ft-base-image:0.9"
-TFT_TOOLS_IMG = "quay.io/wizhao/tft-tools:0.3"
+TFT_TOOLS_IMG = "quay.io/vpunj/tft-tools:0.1-x86_64"
```

Comment:
We should probably keep this consistent / ensure it is a multi-arch image

Reply:
More specifically, if this is working, the github action in this repo should automatically build the x86/arm images, create a manifest, and push it to `quay.io/wizhao/tft-tools:latest`

https://github.com/wizhaoredhat/ocp-traffic-flow-tests/actions/runs/6200895676

Reply:
I second this comment. We should have it multi-arch.

Reply:
sounds good. I'll do that instead.